easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.13.0-foss-2022a.eb

```diff
@@ -0,0 +1,232 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.13.0'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('Bazel', '5.1.1'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.36.0', '-nodocs'),
+    ('pybind11', '2.9.2'),
+    ('UnZip', '6.0'),
+    # System protobuf doesn't seem to work: https://github.com/tensorflow/tensorflow/issues/61593
+    # So don't add it here
+]
+dependencies = [
+    ('Python', '3.10.4'),
+    ('h5py', '3.7.0'),
+    ('cURL', '7.83.0'),
+    ('dill', '0.3.6'),
+    ('double-conversion', '3.2.0'),
+    ('flatbuffers', '2.0.7'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '2.7.1'),
+    ('ICU', '71.1'),
+    ('JsonCpp', '1.9.5'),
+    ('libjpeg-turbo', '2.1.3'),
+    ('NASM', '2.15.05'),
+    ('nsync', '1.25.0'),
+    ('SQLite', '3.38.3'),
+    ('patchelf', '0.15.0'),
+    ('protobuf-python', '4.21.9'),
```

Comment:
This introduces a new variant of `protobuf-python` as dependency in the `2022a` generation of easyconfigs.

We can add an exception in the test suite for this if needed, but is it worth pursuing this since we already have TensorFlow 2.13.0 with `foss/2022b` *and* `foss/2023a`?

Reply:
Not sure. Given I found and worked around the Bazel issue in 2022b+ we could also skip this to avoid possible confusion/mixup with module versions.